### ISSUE TITLE
Clarify assignment 3 - make it clearer

### ISSUE THREAD

**Issue by gilliankerr** [LogicalOutcomes/dhis2-training#6]
Request from ARC 
Look at the worksheet below and copy it... but then the tables in front of me, I was reading the instructions, I didn't understand what I was supposed to do.

**Comment by saragaudon** [comment]
For Pathfinder: (and in general) - we need to get an English writer to revisit the assignments and materials to ensure it's in plain,simple language.


Assign this task to Nicholas / once he's in the GitHUB space.


---
Sara Gaudon | Executive Director, LogicalOutcomes  |  www.logicaloutcomes.net<http://www.logicaloutcomes.net/>
Toronto (647) 478-5634 x103 New York (518) 293-2718  |  sara@logicaloutcomes.net


________________________________

**Comment by gilliankerr** [comment]
Yes, I agree – and not only look at it for English clarity but also test all the steps to ensure they make sense.  (For github issues, remove email signatures and previous email contents so they don’t clutter up the discussion)
g
